src/buck_vs_ldo.tex

```diff
@@ -103,7 +103,9 @@ \subsubsection{Operation}
     \label{fig:buck_converter_states}
 \end{figure}
 
-\noindent Buck converters usually operate in CCM (\textit{Continuous Conduction Mode}) meaning there is constantly current flowing forward in the inductor. In CCM, $D_{on} = \frac{V_{out}}{V_{in}} = \frac{T_{on}}{T_{on} + T_{off}}$, describes the duty cycles of an ideal buck converter as shown in the following figure.
+\noindent Recall from Question \ref{section:passives} that the energy stored within an inductor and capacitor is given by $E = \frac{1}{2} \cdot L \cdot I^2$ and $E = \frac{1}{2} \cdot C \cdot V^2$ respectively. Conceptually when the high side switch is ON, energy is being transferred from the input source to the inductor and capacitor such that when the input source is disconnected these energy storage elements can supply energy to the load. Specifically, when the high side switch is on, the current through the inductor  and the voltage accross the capacitor increases. Consequently, when the high side switch is off the current through the inductor decreases (or stops at zero) and the voltage accross the capacitor decreases.  \newline
```

Comment:
comma after coneptually, disconnected.
extra space after inductor. 
comma after "off" after "when the high side switch is off"